sway/main.c

```diff
@@ -206,7 +206,7 @@ static void drop_permissions(void) {
 		}
 	}
 	if (setuid(0) != -1) {
-		wlr_log(WLR_ERROR, "Root privileges can be restored.");
+		wlr_log(WLR_ERROR, "Root privileges can be restored. Refusing to start.");
```

Comment:
Wouldn't it be clearer to say something like "
```suggestion
		wlr_log(WLR_ERROR, "Dropping root privileges failed. Refusing to start.");
``` ?